test/unit_tests/braket/circuits/test_circuit.py

```diff
@@ -1306,3 +1369,159 @@ def test_diagram(h):
 
     assert h.diagram(mock_diagram) == expected
     mock_diagram.build_diagram.assert_called_with(h)
+
+
+def test_add_parameterized_check_true():
+    theta = FreeParameter("theta")
+    circ = (
+        Circuit()
+        .ry(angle=theta, target=0)
+        .ry(angle=theta, target=1)
+        .ry(angle=theta, target=2)
+        .ry(angle=theta, target=3)
+    )
+    expected = set()
+    expected.add(theta)
+
+    assert circ.parameters == expected
+
+
+def test_add_parameterized_instr_parameterized_circ_check_true():
+    theta = FreeParameter("theta")
```

Comment:
can you also add an alpha2 that is alpha2=FreeParameter("alpha") and add it to the circuit? What is expected for the parameters?

Reply:
Parameters remain unchanged. Adding in this test case.